app/assets/stylesheets/views/_brexit-landing-page.scss

```diff
@@ -58,7 +58,13 @@ $red: #E61E32;
 }
 
 .landing-page__section-title {
-  padding: govuk-spacing(3) govuk-spacing(2) govuk-spacing(3) 0;
+  padding: govuk-spacing(7) govuk-spacing(2) govuk-spacing(2) 0;
+  margin-bottom: 20px;
```

Comment:
@vanitabarrett my dim memory tells me there was a reason this was better as px `govuk-spacing(2)` is that still true?

Reply:
Not sure.. some spacing definitions become smaller on smaller breakpoints, perhaps we wanted this to always be 20px?

Reply:
ok will leave it be for the moment